dockstore-webservice/src/main/java/io/dockstore/webservice/DockstoreWebserviceConfiguration.java

```diff
@@ -1198,4 +1218,19 @@ public void setCheckUrlLambdaVersion(String checkUrlLambdaVersion) {
             this.checkUrlLambdaVersion = checkUrlLambdaVersion;
         }
     }
+
+    public enum DoiAutoGenerationPolicy {
+        /**
+         * Do not auto generate DOIs
+         */
+        NONE,
+        /**
+         * Only generate DOIs for GitHub organizations in {@code getGitHubOrgsWithDoiGeneration}
+         */
+        ALLOW_LIST,
```

Comment:
javadoc is a little unclear, I'm assuming that allow_list'ed organizations can also opt-out?